examples/actix-example.rs

```diff
@@ -1,9 +1,10 @@
-use actix_web::{web, App, HttpResponse, HttpServer, Responder};
+use actix_web::rt::task::JoinHandle;
```

Comment:
Updating testing endpoint to account for structural changes to the test harness.